client/my-sites/site-settings/form-writing.jsx

```diff
@@ -157,7 +175,9 @@ const getFormSettings = partialRight( pick, [
 	'wp_mobile_excerpt',
 	'wp_mobile_featured_images',
 	'wp_mobile_app_promos',
-	'post_by_email_address'
+	'post_by_email_address',
+	'carousel_display_exif',
+	'carousel_background_color'
```

Comment:
There's an issue here: if the `carousel` module is disabled, you're not able to save its settings. Try disabling the Carousel module, and click "Save Settings".

So, several things you'll probably need to do to cover cases where the modules are disabled:

1. Add the module names as settings here.
2. Add the modules and their respective settings in `filterSettingsByActiveModules` in `state/settings/jetpack/utils.js`
3. Add the fields you added to the `filterSettingsByActiveModules` tests in `state/settings/jetpack/test/utils.js`

Also, you should make sure to test saving the Writing form works properly with these modules disabled after you apply these changes.

See #10971 for background.